parser/tools/generate_ast.cc

```diff
@@ -1036,7 +1036,7 @@ void emitNodeClassfile(ostream &out, NodeDef &node) {
                     out << "    fmt::format_to(std::back_inserter(buf), \", \\\"{}\\\"\", " << arg.name
                         << ".toString(gs));\n";
                 } else {
-                    out << "    fmt::format_to(std::back_inserter(buf), \", :\" + JSON::escape(" << arg.name
+                    out << "    fmt::format_to(std::back_inserter(buf), \", :{}\", JSON::escape(" << arg.name
```

Comment:
This caused a fun Clang 12 bug that I couldn't reproduce on a small example, but which fired for this file every time I tried to build.

The thing is: this `operator+` is not consteval, so it was technically an error anyways, and would have had to be fixed like this. but the crazy part is that it somehow crashed clang